process/block/metablock.go

```diff
@@ -481,7 +481,7 @@ func (mp *metaProcessor) verifyCrossShardMiniBlockDstMe(header *block.MetaBlock)
 		return err
 	}
 
-	//if all miniblockshards hashes are in header miniblocks as well
+	// if all miniblockshards hashes are in header miniblocks as well
```

Comment:
on blockprocessor commit you should push all the logs from txLogsProcessor to INDEXER interface at end of each block.

Reply:
done.